x/stakeibc/keeper/hooks.go

```diff
@@ -184,46 +184,42 @@ func (k Keeper) UpdateRedemptionRates(ctx sdk.Context, depositRecords []recordst
 	}
 }
 
-// Determine the undelegated balance from the deposit records queued for staking
-func (k Keeper) GetUndelegatedBalance(hostZone types.HostZone, depositRecords []recordstypes.DepositRecord) sdk.Dec {
-	// filter to only the deposit records for the host zone with status DELEGATION_QUEUE
-	undelegatedDepositRecords := utils.FilterDepositRecords(depositRecords, func(record recordstypes.DepositRecord) (condition bool) {
-		return ((record.Status == recordstypes.DepositRecord_DELEGATION_QUEUE ||
-			record.Status == recordstypes.DepositRecord_DELEGATION_IN_PROGRESS) &&
-			record.HostZoneId == hostZone.ChainId)
-	})
-
-	// sum the amounts of the deposit records
+// Determine the deposit account balance, representing native tokens that have been deposited
+// from liquid stakes, but have not yet been transferred to the host
+func (k Keeper) GetDepositAccountBalance(chainId string, depositRecords []recordstypes.DepositRecord) sdk.Dec {
+	// sum on deposit records with status TRANSFER_QUEUE or TRANSFER_IN_PROGRESS
 	totalAmount := sdkmath.ZeroInt()
-	for _, depositRecord := range undelegatedDepositRecords {
-		totalAmount = totalAmount.Add(depositRecord.Amount)
+	for _, depositRecord := range depositRecords {
+		transferStatus := (depositRecord.Status == recordstypes.DepositRecord_TRANSFER_QUEUE ||
+			depositRecord.Status == recordstypes.DepositRecord_TRANSFER_IN_PROGRESS)
+
+		if depositRecord.HostZoneId == chainId && transferStatus {
```

Comment:
Why do we call it HostZoneId in some places and chainId in most other places? Is this supposed to signify that the deposit is actually on stride but the token is of a type linked to HostZone or is this a naming convention which we have moved away from for new types but it is hard to change the core types?

Reply:
it's just an inconsistency across our codebase that I've been slowly trying to remedy.

You'll often see HostZoneId, HostChainId, ChainId, etc. but they're all the same